src/Template/Element/sort_menu.ctp

```diff
@@ -0,0 +1,35 @@
+<md-menu md-offset="5 50" md-position-mode="target-right target">
+    <md-button ng-click="$mdOpenMenu($event)">
+        <md-icon>sort</md-icon> Sort by
+    </md-button>
+    <md-menu-content ng-cloak>
```

Comment:
You set `ng-cloak` for the menu content, but the sort button and its icons also need to be hidden during page load. See how it looks during page load:

![image](https://user-images.githubusercontent.com/5107734/88481088-c3120b00-cf59-11ea-92a2-381df9a958ff.png)


And then after load:

![image](https://user-images.githubusercontent.com/5107734/88480985-71698080-cf59-11ea-96b1-910a7a10d221.png)

Reply:
I see. Maybe my browser cached the pages so I could not see this detail and the other `ng-cloak` seemed to work enough.

I will move `ng-cloak` in the top `md-menu` and see if one is enough for all.

Reply:
I modified it it should be ok now.